### PR TITLE
Fix loading bar reload texture issue

### PR DESCRIPTION
**Pull request by zilongshanren**

### CODE REVIEW

**Comment by doomsdayer** [comment]
@andyque  I think it's ok.:thumbsup:

File: tests/cpp-tests/Resources/ccs-res [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 615f52a6c5fc1f60c6f01832d110c78a65be3a11
+Subproject commit ce5606d4e520d2671a678b0ba3d5a0b84fdb5ab9
```
Comment (minggo) [inline tests/cpp-tests/Resources/ccs-res:1]: Should update it?

Comment (zilongshanren) [inline tests/cpp-tests/Resources/ccs-res:1]: @minggo 
Yes, I added two new images.

Comment (minggo) [inline tests/cpp-tests/Resources/ccs-res:1]: Ok.

**Comment by minggo** [comment]
[ci rebuild]

**Pull request merged**